src/pg_to_evalscript/javascript_processes/array_append.js

```diff
@@ -0,0 +1,17 @@
+function array_append(arguments) {
+  const { array, value } = arguments;
```

Comment:
`array` should be `data`

also in the test file

Reply:
:point_up:  what he said

Reply:
:point_up:

Reply:
fixed